torch/fx/graph.py

```diff
@@ -451,24 +451,28 @@ def append_stacktrace_summary(node : Node):
                         prev_stacktrace = node.stack_trace
 
                         lines = node.stack_trace.strip().split('\n')
-                        idx = 0
-                        while idx < len(lines):
+                        # stacktrace should have innermost frame last, so we
+                        # iterate backwards to find the first line that starts
+                        # with 'File '
+                        idx = len(lines) - 1
+                        while idx >= 0:
```

Comment:
nit: use a range loop here? `for idx in range(len(lines)-1, -1, -1)`

Reply:
ok, I rewrote to simplify the code.